drivers/ethernet/Kconfig.sam_gmac

```diff
@@ -28,6 +28,27 @@ config ETH_SAM_GMAC_QUEUES
 	  Select the number of hardware queues used by the driver. Packets will be
 	  routed to appropriate queues based on their priority.
 
+config ETH_SAM_GMAC_FORCE_QUEUE
+	bool "Force all traffic to be routed through a specific queue"
+	depends on ETH_SAM_GMAC_QUEUES > 1
+	depends on NET_TC_RX_COUNT < 5
+	help
+	  This option is meant to be used only for debugging. Use it to force all
+	  traffic to be routed through a specific hardware queue. With this enabled
+	  it is easier to verify whether the chosen hardware queue actually works.
+	  This works only if there are four or fewer RX traffic classes enabled, as
+	  the SAM GMAC hardware supports screening up to four traffic classes.
+
+config ETH_SAM_GMAC_FORCED_QUEUE
+	int "Queue to force the packets to"
+	depends on ETH_SAM_GMAC_FORCE_QUEUE
+	default 0
+	range 0 1 if ETH_SAM_GMAC_QUEUES = 2
+	range 0 2 if ETH_SAM_GMAC_QUEUES = 3
```

Comment:
Hello @ulfalizer  - is there any way to make something like that a little bit prettier? With a single range entry instead of two.
Something like `range 0 (ETH_SAM_GMAC_QUEUES-1)`  is what is needed.